### PR TITLE
fix: bare-except

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: bikeshed/boilerplate.py [hunk 1]
```diff
@@ -363,7 +363,7 @@ def disambiguator(ref, types, specs):
     if ref.for_:
         try:
             disambInfo.append("for {}".format(", ".join(x.strip() for x in ref.for_)))
-        except:
+        except ValueError:
```
Comment (nschonni) [inline bikeshed/boilerplate.py:366]: Not sure about this one

File: bikeshed/config/printjson.py [hunk 2]
```diff
@@ -79,4 +79,4 @@ def printjsonprimitive(x):
         return "null"
     if isElement(x):
         return repr(x) + ":" + outerHTML(x)
-    raise Exception(x)
+    raise ValueError(f"Could not print value: {x}")
```
Comment (nschonni) [inline bikeshed/config/printjson.py:82]: Not sure if it's better to have a broad Exception here

**Review by nschonni — COMMENTED** [review]
Not 100% that I didn't miss something here about the broad excepts

File: bikeshed/boilerplate.py [hunk 3]
```diff
@@ -109,7 +109,7 @@ def addSpecVersion(doc):
                     .decode(encoding="utf-8")
                     .strip()
                 )
-        except:
+        except FileNotFoundError:
```
Comment (tabatkins) [inline bikeshed/boilerplate.py:112]: Pretty sure this should be a `subprocess.CalledProcessError` too.

File: bikeshed/boilerplate.py (hunk 1 above)
Comment (tabatkins) [reply bikeshed/boilerplate.py:366]: I have no clue what originally triggered me to add this. Regardless, it doesn't happen anymore; I've just pushed a commit to remove this try/except.

**Review by tabatkins — CHANGES_REQUESTED** [review]
changes in comments

File: bikeshed/config/printjson.py (hunk 2 above)
Comment (tabatkins) [reply bikeshed/config/printjson.py:82]: Nah, seems fine, I was just lazy about summoning the correc type.

File: bikeshed/boilerplate.py (hunk 3 above)
Comment (nschonni) [reply bikeshed/boilerplate.py]: Swapped

**Pull request merged**